steamclog/build.gradle

```diff
@@ -27,28 +27,29 @@ afterEvaluate {
 android {
     compileSdkVersion versions.compileSdk
 
+    publishing {
+        //Publish your app as an AAB
+        singleVariant("release")
+    }
+
     // ADD COMPATIBILITY OPTIONS TO BE COMPATIBLE WITH JAVA 1.8
     // Added for Sentry support
     compileOptions {
-        sourceCompatibility = JavaVersion.VERSION_1_8
-        targetCompatibility = JavaVersion.VERSION_1_8
+        sourceCompatibility = JavaVersion.VERSION_17
+        targetCompatibility = JavaVersion.VERSION_17
     }
 
     defaultConfig {
         minSdkVersion 21
         targetSdkVersion versions.compileSdk
-        versionCode 4
-        versionName "2.4"
         consumerProguardFiles "consumer-rules.pro"
     }
 
     buildTypes {
         debug {
-            debuggable true
             minifyEnabled false
         }
         release {
-            debuggable false
```

Comment:
Just wondering why these where changed? Do you know what the default is?

Reply:
it was removed when I updated gradle. I can add it back in